src/app/pages/test-report/cat-adi-part3/components/adi3-end-test-modal/adi3-end-test-modal.html

```diff
@@ -30,20 +30,25 @@
   </modal-result-item>
   <modal-result-item
     [adi3]="true"
-    [showIcon]="testState > 0"
+    [showIcon]="false"
     [testState]="testState"
     [label]="'Overall score'"
     [score]="totalScore"
     [isPass]="testResult.activityCode === '1'">
   </modal-result-item>
   <modal-result-item
     [adi3]="true"
-    [showIcon]="true"
+    [showIcon]="false"
     [showScore]="false"
     [label]="'Review'"
     [score]="0"
     [isPass]="!!feedback">
   </modal-result-item>
+  <ion-row>
+      <ion-col>
+        <p class="ion-padding ion-justify-content-center" *ngIf="!isValidDashboard">Please complete all sections of the test report to continue to debrief</p>
+      </ion-col>
+  </ion-row>
   <ion-row class="ion-justify-content-center">
     <ion-col size="84">
       <ion-button *ngIf="!!isValidDashboard || isTestReportPopulated" class="mes-primary-button" (click)="onContinue()">
```

Comment:
We should do 
```
*ngIf="!!isValidDashboard || isTestReportPopulated; else missingSections;"
```

Then 
```
<ng-template #missingSections>
 .. whatever you as the else path
</ng-template>
```
This the guarantees only one section displays. You are currently not using both of the conditions in the top if so conceivably both could appear